### ISSUE TITLE
Building Linux copy breaks with Unknown identifier : path

### ISSUE THREAD

**Issue by Mia-Rain** [FunkinCrew/Funkin#74]
I'm trying to build a fresh copy of this for linux, since the version on ITCH, is abt 20 days old, and also uses glibc 2.29, where my old-ish install uses glibc 2.27 (Older and newer versions don't work together because glibc is dumb). I followed #66 on setting it up, and it almost works, but I'm getting the following error, any idea how I could fix this? 
```
$ lime build linux -release
/usr/share/haxe/lib/newgrounds/1,1,2/lib/Source/io/newgrounds/NG.hx:239: characters 38-42 : Unknown identifier : path
/usr/share/haxe/lib/newgrounds/1,1,2/lib/Source/io/newgrounds/NG.hx:239: characters 38-42 : For optional function argument 'args'
```

**Comment by ElijahZAwesome** [comment]
The newgrounds haxe package is broken. You'll have to either replace that `args` bit with `path` in the original file the error occurs in, or install the latest source which has fixed this issue:

```
wget -O newgrounds.zip https://codeload.github.com/Geokureli/Newgrounds/zip/master
haxelib install newgrounds.zip
```

**Comment by gedehari** [comment]
> The newgrounds haxe package is broken. You'll have to either replace that `args` bit with `path` in the original file the error occurs in, or install the latest source which has fixed this issue:

Ahh yeah, from the very first time I tried to build on Linux this happened, now I know why. Thankies.

**Comment by ThatGeekyWeeb** [comment]
@ElijahZAwesome, Thanks a ton, I'll try that when I get home :)

**Comment by ThatGeekyWeeb** [comment]
Note that it has to be executed from the `export/release/**/bin` (You must be in the folder on linux (`cd export/release/linux/bin`)), otherwise it won't launch.

**Comment by ninjamuffin99** [comment]
ah yep, i noticed this too! I made a PR to the newgrounds haxe repository and it got accepted, but it's not in the Haxelib yet!

https://github.com/Geokureli/Newgrounds/pull/46

Shout at @geokureli to update it!

meanwhile temp fix is to install the Newgrounds library right from the Github
`lime git newgrounds https://github.com/geokureli/newgrounds.git`

**Comment by ninjamuffin99** [comment]
closing since it's an issue with another library, idk if that's what im supposed to do, but im gonna do it anyways!

**Comment by Geokureli** [comment]
I have been summoned. Sorry my lib is messing with your fanbase, cam. on it

**Comment by Geokureli** [comment]
released newgrounds 1.1.3

**Comment by ninjamuffin99** [comment]
ballin
